src/board/system76/common/battery.c

```diff
@@ -75,6 +75,23 @@ int16_t battery_charger_configure(void) {
     return battery_charger_disable();
 }
 
+void battery_limit_capacity(void) {
+    // Clevo requires limiting charge to 90% of the SBS reported capacity.
+    battery_info.full_capacity = ((uint32_t)battery_info.full_capacity * 90) / 100;
```

Comment:
It is probably necessary to also adjust the design capacity down.

Reply:
Potentially. Linux uses it as a fallback if it considers `FullChargeCapacity()` invalid.

And in turn, `RemainingCapacityAlarm()` would probably have to be changed as well.